deploy/deploy.sh

```diff
@@ -16,12 +16,6 @@ source ../.secrets/postgres.env
 IMAGE=quay.io/ciam_authz/insights-rebac
 IMAGE_TAG=latest
 
-# Prepare bonfire env
```

Comment:
it gives me message:
```
bonfire needs to be installed
```

but maybe we can put it in **prerequisites** in readme ?

Reply:
(which is basically already there :)